lib/cookiesalert/version.rb

```diff
@@ -1,3 +1,3 @@
 module Cookiesalert
-  VERSION = "0.1.2"
+  VERSION = "1.0.0"
```

Comment:
I would stay beta until we have thoroughly tested its use.
```suggestion
  VERSION = "0.2.0"
```